### DISCUSSION TITLE
Netbox on Red Hat Enterprise Linux release 8.7

### DISCUSSION THREAD

**Discussion by berkeleycowboy** [netbox-community/netbox#12888] [Q&A]
Has anybody had any success getting Netbox on Red Hat Linux.  

Did not have any issues installing Netbox on Centos OS.  

My pain point is the DJango installation.

**Comment by candlerb** [comment]
The answer has not changed from when you asked the same question 4 days ago - #12854

There is no Django to install.  The Netbox `./upgrade.sh` script installs all its dependencies, including Django, inside a python virtualenv.

Therefore, if you are having a problem, then please start by showing the error you see, and the command you typed that caused that error to appear. There is good advice on how to approach troubleshooting in general [here](http://www.catb.org/~esr/faqs/smart-questions.html#intro).

**Reply by berkeleycowboy** [reply]
OK,  I will start a new thread.

**Comment by berkeleycowboy** [comment]
Thank you for your reply.  
I'm going to perform the installation again so can show where the installation fails.

**Comment by berkeleycowboy** [comment]
Performed a clean of Netbox.  PostgreSQL and Redis installed and validated successfully. 

The netbox upgrade appears to run successfully but there are many errors errors at the tail end of the install.  

The "sudo PYTHON=/usr/bin/python3.9 /opt/netbox/upgrade.sh" command is then executed and again their appears to be errors at the tail end of the script.  

This source command is then executed generating the following output.  

[weikeld@scdl-ifipam-001 ~]$ source /opt/netbox/venv/bin/activate
-bash: /opt/netbox/venv/bin/activate: Permission denied
[weikeld@scdl-ifipam-001 ~]$ sudo source /opt/netbox/venv/bin/activate
[sudo] password for weikeld: 
sudo: source: command not found
[weikeld@scdl-ifipam-001 ~]$ 

Can you let me know if the errors are expected or something was missed.  

Thanks in advance.





[Netbox installation failure.docx](https://github.com/netbox-community/netbox/files/11795211/Netbox.installation.failure.docx)
